cg/src/domains/management/subjects/subjects.controller.js

```diff
@@ -1,12 +1,48 @@
 const SubjectsService = require('./subjects.service');
 
+const PAGINATION_COUNT = 2; // HARD CODED CONSTANT
```

Comment:
This variable should be renamed PAGE_SIZE, also 2 is too small

Reply:
It's small to facilitate tests, I will change it in the final version, thanks for the reminder o/